packages/terra-consumer-nav/CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 ChangeLog
 =========
 
+# 0.1.0-BETA.3 - (--)
```

Comment:
I think you are technically correct. But i'm not sure the date matters that much. I'll work with @bjankord to see what his process is to make things easiest